app/controllers/users_controller.rb

```diff
@@ -1,3 +1,9 @@
 class UsersController < ApplicationController
-  expose :users, -> { User.all }
+  expose :user
+  expose_decorated(:users, -> { User.all })
```

Comment:
Yes, u have :) It makes the controller more complicated in the case of custom logic. Yes, it's much simpler to start but what I learned to from my experience, in the later stage it can be very hard to use it parallel with your custom crud logic. This force you to not using the `expose` method which in fact means removing this gem.

If you want, use it... I just wanted to give you a tip.

BTW.
It doesn't matter who show you this gem. It matters, why you use it.